tasks/setup-Debian.yml

```diff
@@ -13,6 +13,7 @@
       - ca-certificates
       - gnupg2
     state: present
+  when: offline_mode == '1'
```

Comment:
At this point, your offline install mode is already assuming a Red Hat derivative and we don't use any Debian derivative anywhere in C3 so you may as well remove entirely Debian-specific code.

Reply:
Ok, I'll delete the debian part.